sport/app/football/views/matchStats/matchStatsPage.scala.html

```diff
@@ -3,8 +3,7 @@
 @import implicits.Football._
 @import views.FootballHelpers._
 
-
-@(page: MatchPage, competition: Option[model.Competition])(implicit request: RequestHeader, context: model.ApplicationContext)
+@(page: _root_.football.controllers.MatchPage, competition: Option[model.Competition])(implicit request: RequestHeader, context: model.ApplicationContext)
```

Comment:
You are right. It is not necessary.
However IntelliJ is confused and pick up the `football` package from another app (admin). That's why I originally added `_root_`. 
I removed it and will try to fix IntelliJ instead